docs/pages/brownfield/integration-with-existing-apps.mdx

```diff
@@ -0,0 +1,644 @@
+---
+title: Integrating a React Native view into existing native apps
+sidebar_title: Integrated approach
+description: Add Expo and React Native to existing native apps using the "integrated" approach.
+hideTOC: true
+---
+
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+
+React Native offers a flexible development strategy. Whether you're launching a new app or looking to integrate an existing native app, it provides a powerful solution. You can leverage its capabilities to introduce new features or views incrementally, allowing your team to adopt modern, cross-platform development without a complete rewrite.
+
+This guide will walk you through the steps to add a React Native view into an existing native app using the "integrated" approach.
+
+## Prerequisites
+
+To integrate React Native into your existing application, you'll need to set up a JavaScript development environment. This includes installing Node.js to run the Expo CLI and Yarn to manage the project's JavaScript dependencies.
+
+
+- [Node.js (LTS)](https://nodejs.org/en/): The runtime to execute JavaScript code and Expo CLI.
+- [Yarn](https://yarnpkg.com/): A package manager for installing and managing JavaScript dependencies.
```

Comment:
```suggestion
- [Node.js (LTS)](https://nodejs.org/en/): The runtime to execute JavaScript code and Expo CLI.
- [Yarn](https://yarnpkg.com/): A package manager for installing and managing JavaScript dependencies.
- **[iOS only]** [CocoaPods](https://cocoapods.org/): one of the dependency management system available for iOS. CocoaPods is a Ruby [gem](https://en.wikipedia.org/wiki/RubyGems). You can install CocoaPods using the version of Ruby that ships with the latest version of macOS.

Learn more from the [Set up environment guide](/get-started/set-up-your-environment/).
```

add cocoapods and link to our setup environment guide
cc @amandeepmittal if you know we have any convention to represent the `[iOS only]` part

Reply:
We do have `PlatformTag` that we can use here:

```md
- <PlatformTags platforms={['ios']} /> [CocoaPods](https://cocoapods.org/): one of the dependency
  management system available for iOS. CocoaPods is a Ruby
  [gem](https://en.wikipedia.org/wiki/RubyGems). You can install CocoaPods using the version of Ruby
  that ships with the latest version of macOS.
```

This will render as shown below and include the Platform.

  
<img width="1086" height="282" alt="CleanShot 2025-09-05 at 19 08 35" src="https://github.com/user-attachments/assets/c40c710e-ca48-4665-bb8d-82eb40776f53" />